recipes/Orange3-SingleCell/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "Orange3-SingleCell" %}
+{% set version = "0.8.1" %}
+{% set sha256 = "9eed302365948722b87502aca34fb19a037e6fd9a356f9d2dddb666e6dcd7c6c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
```

Comment:
Is this project a pure Python project?
If it is you can add ``noarch::python``, remove the ``  skip: True  # [py2k]`` and in ``requirements`` you have to put ``- python >2``